pkg/controller/cloneset/utils/cloneset_utils.go

```diff
@@ -94,15 +94,26 @@ func IsRunningAndAvailable(pod *v1.Pod, minReadySeconds int32) bool {
 	return pod.Status.Phase == v1.PodRunning && podutil.IsPodAvailable(pod, minReadySeconds, metav1.Now())
 }
 
-// GetUpdatedPods returns Pods already reach latest revision
-func GetUpdatedPods(pods []*v1.Pod, updateRevision string) []*v1.Pod {
-	var updated []*v1.Pod
+// GetPodsMatchedRevision returns Pods matched the given revision
```

Comment:
Can we merge GetPodsMatchedRevision and GetPodsMatchedRevision ? e.g., called SplitPodsByRevision

Reply:
done